roles/backend/handlers/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Is this file required with the default content?

Reply:
Fixed https://github.com/3scale/saas-operator/pull/6/commits/afa2b8f848475552b244434597e18be9e1cda422